.env.example

```diff
@@ -1,4 +0,0 @@
-SKIP_PREFLIGHT_CHECK=true
```

Comment:
Please, don't remove this file, it is an example

Reply:
Added .env.example